draft-ietf-quic-http.md

```diff
@@ -1381,8 +1381,11 @@ sent, and receipt MAY be treated as an error of type H3_FRAME_UNEXPECTED.
 QUIC allows the application to abruptly terminate (reset) individual streams or
 the entire connection when an error is encountered.  These are referred to as
 "stream errors" or "connection errors" and are described in more detail in
-{{QUIC-TRANSPORT}}.  An endpoint MAY choose to treat a stream error as a
-connection error.
+{{QUIC-TRANSPORT}}.
+
+An endpoint MAY choose to treat a stream error as a connection error under
+certain circumstances.  Implementations need to consider the impact on
+outstanding requests before making this choice.
```

Comment:
LG. 

The other impact of closing conenction with an error relates to Alt-Svc. A client maintaining an Alt-Svc cache may interpret a connetion error as a broken service, which could cause a fallback (e.g. to HTTP/2) and a failure to switch to the alternative until the black mark expires. 

We saw _something_ like this between Chrome and Cloudflare, where one of us was mistakenly detecting something as an error. @RyanAtGoogle might recall.

Reply:
In that case, I believe Chrome was sending a RESET_STREAM after the stream had been closed, which is technically legal, but unnecessary.  And your implementation immediately closed the connection in response to a RESET_STREAM on an already closed stream, causing Chrome to mark QUIC as 'broken' for 5 minutes.

Reply:
That case smells like a server bug.  You should not treat receipt of a legal frame as an error.  I assume that the RESET_STREAM was sent only in the case that the stream was fully acknowledged.  Otherwise that might have been a genuine problem, though still not cause for termination.